tools/github/contributions.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+parse_args() {
+    if [ -z "$2" ]; then
+        echo "value for parameter '$1' cannot be empty"
+        exit 1
+    fi
```

Comment:
Could remove this based on my comment below, in case you ever want boolean args supported